frontend/next.config.js

```diff
@@ -7,7 +7,8 @@ const isProdBuild = process.env.NODE_ENV === nodeEnv.PRODUCTION;
 
 const PLAUSIBLE_URL = "https://plausible.io";
 
-const SCRIPT_SRC = ["'self'", PLAUSIBLE_URL];
+// unsafe-eval is required for next-mdx-remote. Currently pending sec review
+const SCRIPT_SRC = ["'self'", "'unsafe-eval'", PLAUSIBLE_URL];
```

Comment:
@jakeyheath is it okay to use `unsafe-eval` here? My impression is that this opens up to XSS, but not sure if it's a valid concern in our context 😆 Thank you!

Reply:
https://chanzuckerbergteam.slack.com/archives/C02H8TRBPC7/p1652731472038849

Reply:
sorry I am not sure if yall got your answer here. last week, we were having our central all hands so I was a bit AFK. Ideally, I wouldn't want to see this, but sounds like there isn't a way around this library. They specifically call it out as a requirement 🤷🏻 . 

Generally, CSP is a defense-in-depth. You are already protected from XSS by using React.  Try not to use dangerouslySetInnerHTML and you should be good. Maybe periodically review that library to see if they remove that need for unsafe-eval as a requirement.

Reply:
Thanks @jakeyheath for your detailed answer! We do have another option to use (https://nextjs.org/docs/advanced-features/using-mdx), so maybe it's worth investigating it further?

@seve what's the eng overhead for using@next/mdx again vs. the hashicorp one?

So far I see it's the part that we need to inject doc site layout to every MDX files. Is there anything else?

Thank you!

Reply:
Also [Data Portal scExpression](https://cellxgene.cziscience.com/scExpression) uses a third-party library ([echarts](https://github.com/apache/echarts)) to render data viz DOM elements, so it's outside of React's control too!